pkg/config/config.go

```diff
@@ -461,9 +464,12 @@ Compatible cuDNN version is: %s`,
 				return err
 			}
 			console.Debugf("Setting CUDA to version %s from Torch version", c.Build.CUDA)
-		} else if !slices.ContainsString(torchCUDAs, c.Build.CUDA) {
+		} else if len(slices.FilterString(torchCUDAs, func(torchCUDA string) bool { return version.EqualMinor(torchCUDA, c.Build.CUDA) })) == 0 {
```

Comment:
I had trouble understanding this conditional. Could we make this an `else` and break this up over a couple lines?

Reply:
Fixed!